src/scenes/simple-scene.js

```diff
@@ -18,32 +17,43 @@ export class SimpleScene extends Phaser.Scene {
 
   preload() {
 
-    this.load.image('cokecan', 'assets/cokecan.png');
-    this.load.image('ground', 'assets/ground.png');
+    this.load.multiatlas('robot', 'assets/player/robot_walking.json', 'assets');
+    this.load.image('tiles', 'assets/environment/third_party/basic_tileset_144x48.png');
+    this.load.tilemapTiledJSON("map", "assets/environment/basic_horiz_map.json");
+
   }
 
   create() {
 
-    var newWeapon = this.weapons.add(30, 'bullet');
-    console.log(newWeapon);
-
     this.keys = {
       up: this.input.keyboard.addKey(Phaser.Input.Keyboard.KeyCodes.UP),
       left: this.input.keyboard.addKey(Phaser.Input.Keyboard.KeyCodes.LEFT),
       right: this.input.keyboard.addKey(Phaser.Input.Keyboard.KeyCodes.RIGHT),
       down: this.input.keyboard.addKey(Phaser.Input.Keyboard.KeyCodes.DOWN)
     };
 
+    const map = this.make.tilemap({ key: "map"});
+    const tileset = map.addTilesetImage("basic_48_tiles", "tiles");
+    const belowLayer = map.createStaticLayer("background", tileset, 0, 0);
+    const worldLayer = map.createStaticLayer("solid", tileset, 0, 0);
+
     this.player =  new Player({
       scene: this,
```

Comment:
Something feels weird here.  The Scene creates a player and passes itself as an argument and then the Player adds a sprite to the scene.

I think it's the obfuscation of adding something to the scene.